understanding/20/unusual-words.html

```diff
@@ -8,6 +8,14 @@
 <body>
    <h1>Understanding Unusual Words</h1>
    
+   <section id="brief">
+      <h2>In brief</h2>
+      <dl>
+         <dt>Objective</dt><dd>Users can identify and learn what unusual words mean</dd>
+         <dt>Author task</dt><dd>For technical jargon and unusual terms, definitions are available to users</dd>
```

Comment:
Would be more consistent to use command tense here:

```suggestion
         <dt>Author task</dt><dd>Provide definitions for technical jargon and unusual terms</dd>
```